src/odoc/bin/main.ml

```diff
@@ -714,19 +730,38 @@ module Odoc_html_args = struct
     Arg.(
       value & opt_all convert_fpath [] & info [ "asset" ] ~doc ~docv:"file.ext")
 
+  let source_root =
+    let doc =
+      "Source code root for the compilation unit. It must have been compiled \
+       with --source-parent passed."
```

Comment:
I feel it might help to add some explanation what is a "source code root".
```suggestion
      "Source code root for the compilation unit. Used to find the source file \
       from the value of --source-name it was compiled with. Incompatible with \
       --source-file."
```
Also, [line 718](https://github.com/ocaml/odoc/pull/982/files#diff-47026f52e88243e29f89adcdd872b0e32fe9f2368df000cc0db39c986e05b111R718) it should be `--source-parent-file` instead of `--source-parent` (or maybe "`--source-parent-file` and `--source-name` to be more explicit)